### ISSUE TITLE
Template from SampleType need to contain attributes' constrains (actually, it is not a new issue, but can't find old ne any more)

### ISSUE THREAD

**Issue by OlgaKrebs** [seek4science/seek#1075]
Creating SampleType one can define attribute, its type (string, float, date,etc) or constrain values (controlled vocabularies, ontology term,etc). But corresponding downloaded template does only contail attribute names, so user has to look up in SEEK which values are allowed and typos are not prevented

**Comment by floradanna** [comment]
Having the CV as dropdown in Excel used to work, so I guess this is a bug (something broke).
I would say that this is a rather urgent issue.

**Comment by OlgaKrebs** [comment]
yes, I also remember to have seen this... but not sure it was on FAIRDOMHub (or any other testing instance

**Comment by floradanna** [comment]
@OlgaKrebs I have just tried to download a sample type and I actually see the CV in excel. 

Is it still not working for you? Could you please try again and let me know?

<img width="590" alt="Screenshot 2022-07-14 at 10 03 09" src="https://user-images.githubusercontent.com/51486716/178933003-171e9648-889b-43b6-85d6-d67b80143542.png">

**Comment by OlgaKrebs** [comment]
no, it doesn't  

sampleType  173  has 3 columns as CVs:


Tissue | Controlled Vocabulary - BRENDA tissue | Tissue type | - | -
ethnicity | Controlled Vocabulary - Ethnicity | Not specified | - | -
gender | Controlled Vocabulary - Liver cell types22



</html>Tissue	Controlled Vocabulary - [BRENDA tissue](https://fairdomhub.org/sample_controlled_vocabs/19)	Tissue type	-	-
ethnicity	Controlled Vocabulary - [Ethnicity](https://fairdomhub.org/sample_controlled_vocabs/8)	Not specified	-	-
gender	Controlled Vocabulary - [Liver cell types22](https://fairdomhub.org/sample_controlled_vocabs/20)

but exported excel looks like this 
![image](https://user-images.githubusercontent.com/55085007/178976912-d086d72e-0157-42b6-b316-57f7833dbb2a.png)

**Comment by OlgaKrebs** [comment]
no drop downs

**Comment by OlgaKrebs** [comment]
@floradanna which one did you use?

**Comment by OlgaKrebs** [comment]
thanks to Flora - I have to double click on the CVs field in Excel and drop down list appears . May be we need to add a line for users, otherwise they can even start typing in this field and gets error than ...
